.github/workflows/pr-checks.yml

```diff
@@ -81,5 +81,5 @@ jobs:
             - name: Run PHP tests
               run: |
                   mysql -uroot -h127.0.0.1 -e 'SELECT version()' \
-                  && ./bin/install-wp-tests.sh --recreate-db wordpress_test root '' > /dev/null \
+                  && ./bin/install-wp-tests.sh --wp-version=trunk --recreate-db wordpress_test root ''  > /dev/null \
```

Comment:
Interesting that it didn't work on `nightly`; it worked here https://github.com/WordPress/create-block-theme/pull/526/commits/420ad20ae94d58257255ce2d9017abea095e68a1

Reply:
Yes!  Very strange!  I didn't expect it to, I just noticed the || in the code and gave it a whirl.

It made zero sense but I ran with it!